<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamMemberView</class>
 <widget class="QWidget" name="TeamMemberView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>1028</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeamMemberView</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0,1">
   <item>
    <widget class="QGroupBox" name="addRemoveGroupBox">
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="addRemoveLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>75</width>
          <height>55</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="font">
         <font>
          <family>Academy Engraved LET</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="list-remove">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>75</width>
          <height>55</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="membersGroupBox">
     <property name="title">
      <string>Members</string>
     </property>
     <layout class="QVBoxLayout" name="membersLayout">
      <item>
       <widget class="QListView" name="membersListView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
